src/HeadCollector.js

```diff
@@ -1,25 +1,18 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import { Provider } from './headTagsContext';
 
 export default class HeadCollector extends Component {
   static propTypes = {
     headTags: PropTypes.array.isRequired, // eslint-disable-line react/forbid-prop-types
     children: PropTypes.node.isRequired,
   };
 
-  static childContextTypes = {
-    reactHeadTags: PropTypes.object,
+  headTags = {
+    add: tag => this.props.headTags.push(tag),
   };
 
-  getChildContext() {
-    return {
-      reactHeadTags: {
-        add: c => this.props.headTags.push(c),
-      },
-    };
-  }
-
   render() {
-    return React.Children.only(this.props.children);
+    return <Provider value={this.headTags}>{React.Children.only(this.props.children)}</Provider>;
```

Comment:
I think we should use default prettier 80 length.

Reply:
Sure, that's fine, please do in a separate PR though